src/app/animals/animal-list/component.spec.ts

```diff
@@ -1,14 +1,12 @@
 import { Component, Input } from '@angular/core';
 import { TestBed, async } from '@angular/core/testing';
-import { RouterTestingModule } from '@angular/router/testing';
 import { By } from '@angular/platform-browser';
 import { AnimalListComponent } from './component';
 import { CoreModule } from '../../core/module';
 import { AnimalType } from '../model';
 
 @Component({ selector: 'zoo-animal', template: '' })
 class MockAnimalComponent {
-  @Input() key: string
```

Comment:
This is needed - without it the unit tests fail because `AnimalList` is binding stuff to this property.